Expensify/App
# Added Hot reload on desktop platform

@roryabraham Please review.

### Details
Added Hot-reload for the desktop platform. Changes are as follows:
1. Updated desktop dev build command.
2. Added `electron-reloader` package which only watches the electron main process src files.
3. Now WDS(Webpack dev server) is building the code for the Renderer process with HMR(Hot module reload) (Faster than the `watchRenderer` option on `electron-reloader` as it clears the cache on every change), Hooked its end URL with the electron window which will update automatically.


### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes #1466 

### Tests
1. Run `npm run electron:HMR`
2. Let the app load in the electron window.
3. Update any source file in the src folder, the app should hot-reload the changes.
4. Change any file in the desktop folder, the app window should reload.


### Tested On


### Screenshots
 N/A
